src/web/WebStatusService.cpp

```diff
@@ -166,9 +166,9 @@ void WebStatusService::webStatusService(AsyncWebServerRequest * request) {
     if (Mqtt::enabled()) {
         statJson       = statsJson.createNestedObject();
         statJson["id"] = 5;
-        statJson["s"]  = Mqtt::publish_count();
+        statJson["s"]  = Mqtt::publish_count() - Mqtt::publish_fails();
```

Comment:
nice!

Reply:
statJson["s"] is for success, publish_count() is all messages. Same correction for quality calculation.